.storybook/themes/commerce.scss

```
@@ -3,43 +3,589 @@
+		// 18px
+		"font-size-6": 1.1875rem,
+		// 19px
+		"font-size-7": 1.25rem,
+		// 20px
+		"font-size-8": 1.375rem,
+		// 22px
+		"font-size-9": 1.5rem,
+		// 24px
+		"font-size-10": 1.75rem,
+		// 28px
+		"font-size-11": 2rem,
+		// 32px
+		"font-size-12": 2.25rem,
+		// 36px
+		"font-size-13": 2.56rem,
+		// 41px
+		"font-size-14": 3rem,
+		// 48px
+		"font-size-15": 3.25rem,
+		// 52px
+		"line-height-1": 0.875rem,
+		// 14px
+		"line-height-2": 1rem,
+		// 16px
+		"line-height-3": 1.25rem,
+		// 20px
+		"line-height-4": 1.5rem,
+		// 24px
+		"line-height-5": 2rem,
+		// 32px
+		"line-height-6": 2.125rem,
+		// 34px
+		"line-height-7": 2.5rem,
+		// 40px
+		"line-height-8": 3rem,
+		// 48px
+		"line-height-9": 3.5rem,
+		// 56px
+		"line-height-10": 3.875rem,
+		// 62px
+		"box-shadow-1": 0px 0px 16px rgba(25, 25, 25, 0.4),
+		"z-index-1": 1,
+		"z-index-2": 10,
+		"z-index-3": 50,
+		"z-index-4": 100,
+		"z-index-5": 500,
+		"border-radius-1": 0.25em,
+	),
+	$alias: (
+		"layout-max-width": 1440,
+		"layout-content-max-width": 1216,
+		//
+		// --layout-content-scale-width identifies the inner document width for header and content
+		// alignment.  It accounts for a minimum width that has (--global-spacing-5 / 2) margins
+		//
+		"layout-content-scale-width":
+			min(
+				calc(100vw - var(--global-spacing-5)),
+				calc(var(--layout-content-max-width) / var(--layout-max-width) * 100%)
+			),
+		"primary-color": var(--global-neutral-10),
+		"primary-font-family": "Inter",
+		"secondary-font-color": "Lora",
+	),
 	$tokens: (
 		default: (
-			components: (
-				stack: (
-					gap: 1rem,
-				),
-				stack-divider-vertical: (
-					border: none,
-					border-bottom-width: 2px,
-					border-bottom-style: solid,
-					border-bottom-color: #712d2d,
-					height: 1px,
+			// DESIGN ASSUMPTION: Attribution does not require component styling.
+			components:
+				(
+					body: (
+						background-color: var(--global-color-white),
+						color: var(--global-color-black),
+						font-family: var(--primary-font-family),
+						font-size: var(--global-font-size-4),
+						line-height: var(--global-line-height-4),
+					),
+					button: (
+						border: 1px solid transparent,
+						border-radius: var(--global-border-radius-1),
+						font-size: var(--global-font-size-3),
+						line-height: var(--global-line-height-3),
+						font-weight: var(--global-font-weight-7),
+						gap: var(--global-spacing-2),
+					),
+					button-small: (
+						padding: var(--global-spacing-1) var(--global-spacing-3),
+					),
+					button-medium: (
+						padding: var(--global-spacing-3),
+					),
+					button-large: (
+						padding: var(--global-spacing-3),
+						font-size: var(--global-font-size-4),
+						line-height: var(--global-line-height-4),
+					),
+					// DESIGN QUESTION:
+					// Please confirm this style in Storybook for a link button.
+					button-link:
+						(
+							background-color: transparent,
+							color: var(--global-neutral-10),
```

```suggestion
							color: var(--primary-color),
```

looks like we should be using the alias token here, right? https://github.com/WPMedia/arc-themes-components/pull/84#discussion_r859563348

---

made a pr making this fix throughout the style